data/World/Overworld.json

```diff
@@ -1016,11 +1016,11 @@
             "Sheik in Kakariko": "
                 is_adult and Forest_Medallion and Fire_Medallion and Water_Medallion",
             "Kak Anju as Adult": "is_adult and at_day",
-            "Kak Anju as Child": "is_child and at_day",
+            "Kak Anju as Child": "is_child and at_day and can_bonk_crate",
```

Comment:
This should be in logic without the crate if 6 or fewer cuccos are required. I'm not sure if `chicken_count < 7` will work as an expression though.

Reply:
Just checked and can confirm that it works as expected.

Reply:
I added this in, thanks!